mod.ts

```diff
@@ -191,27 +197,30 @@ export class RhumRunner {
    *     The object containing the member to stub.
    * @param string member
    *     The member to stub.
-   * @param any value
+   * @param unknown value
    *     The return value of the stubbed member.
    *
    * @return this
    *     Return this so that stub() calls can be chained.
    */
-  public stub(obj: any, member: string, value: any): this {
+  public stub<T>(obj: Stubed<T>, member: keyof T, value: unknown): this {
```

Comment:
Stubs should work like mocks instead, something like
```ts
const stubedServer: Stubed<Server> = new Stubed(new Server());
stubedServer.stub(methodName, ...);
stubedServer.stub(methodName, ...);
```

Reply:
so the reason why I wrote `stub` with this kind of signature was for ease of use. the team and i develop the drash land projects with syntax in mind first and build around that syntax. this syntax introduces something that we didn't agree on. I'm not saying that it's wrong. it definitely seems more robust, but i think it adds more work. also, we didn't agree on stubs working like mocks. all this being said, it is open to discussion. however, let's keep in mind we already released v1.0. breaking changes would cause us to bump up to v2.0. if we HAVE to bump up to v2.0, then i guess we can't get around that and i'll be fine with it. i do want this to be in release v1.1.x though.

Reply:
we still need to have a discussion about this syntax. think you can make your case here or in the discord channel?